input/parser_test.go

```diff
@@ -1,105 +0,0 @@
-package input
-
-import "testing"
-
-/*
- * RFC5424 parser tests
- */
-
-func Test_SuccessfulRFC5424Parsing(t *testing.T) {
-	p := NewRFC5424Parser()
-
-	tests := []struct {
-		message  string
-		expected RFC5424Message
-	}{
-		{
-			message:  "<134>1 2003-08-24T05:14:15.000003-07:00 ubuntu sshd 1999 - password accepted",
```

Comment:
Obviously we must have these tests back in at some point. When are you planning to do that?